integration_tests/__tests__/version.test.js

```diff
@@ -4,13 +4,15 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
```

Comment:
yeah! flowtyping tests gives a lot of signal and has been very helpful for me in the past. I don't really see any reason not to type them, except the fact that there's going to be a lot of `// $FlowFixMe` for the things where we intentionally break things :D (like `Array.prototype[Symbol.iterator] = null`